docs/src/modules/java/pages/proto.adoc

```diff
@@ -96,6 +109,19 @@ include::example$java-valueentity-shopping-cart/src/main/proto/shoppingcart/shop
 ----
 
 
+To use a multi-field key, add the `entity_key` notation to each field. For
+example, if the service API proto file `SomeMessage` element defines both
+`first_field` and `second_field` as part of the key, do the same for the domain
+message:
+
+[source,proto]
+----
+message SomeMessage {
+    string first_field = 1 [(akkaserverless.field).entity_key = true];
+    string second_field = 2 [(akkaserverless.field).entity_key = true];
+----
+
+
```

Comment:
Entity keys are not defined on domain messages. This part can be removed.

```suggestion
```

Reply:
Done, thanks!